frontend/src/pages/_events/slug/register/RegistrationBottomBar/index.js

```diff
@@ -77,6 +82,9 @@ const RegistrationBottomBar = ({
                 </Button>
             )}
             <Box className={classes.right}>
+                <Typography className={classes.sectionsNum}>
+                    {`${sectionsInfo.index}/${sectionsInfo.sections}`}
+                </Typography>
```

Comment:
*Important*
It would be better if you add a verification to make sure `sectionsInfo` exist, so if 'RegistrationBottomBar' is implemented in another context, where it doesn't get passed the `sectionsInfo` prop, then the component would throw an exception

An example would be:

`{sectionsInfo && (<Typography...}`